config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -2274,6 +2274,21 @@ presubmits:
     always_run: false
     cluster: security
     context: pull-security-kubernetes-bazel-build-canary
+    decorate: true
+    decoration_config:
+      gcs_configuration:
+        bucket: kubernetes-jenkins
+        default_org: kubernetes
+        default_repo: kubernetes
+        path_strategy: legacy
+      gcs_credentials_secret: service-account
+      grace_period: 15000000000
+      timeout: 7200000000000
+      utility_images:
+        clonerefs: registry.svc.ci.openshift.org/ci/clonerefs:latest
```

Comment:
Are y'all not pushing your own yet? Be warned my images will be stale until I'm back in the sates and can figure out how to build stuff using `bazel`...

Reply:
ah lol, let me update the default